app/services/folio/locations.rb

```diff
@@ -13,6 +13,53 @@ def self.label(code:)
       instance.data.dig(code, 'discoveryDisplayName')
     end
 
+    # Output a CSV of all locations, flattening objects and converting UUIDs to codes
+    # @return [String] comma-separated data for all locations
+    def self.to_csv
+      headers = %w[institutionCode campusCode libraryCode primaryServicePointCode servicePointCodes]
+
+      data = instance.data.each_value.map do |location|
+        # flatten details & metadata
+        location.merge!(location['details'] || {})
```

Comment:
Since details is a variable-length field (in that the number of columns will change as we add new types of details), do you think we should put it at the end? At least that way, we could use consistent formatting + styles for all the other stuff